task/01-strings-tasks.js

```diff
@@ -69,7 +69,9 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
 function extractNameFromTemplate(value) {
-    throw new Error('Not implemented');
+    let temp = value.replace(' ', '');
+    let ln = 'Hello,'.length;
+    return temp.substring(ln, temp.length - 1);
 }
```

Comment:
looks good, but please note substring can work with negative numbers also to take symbols from end. In the case `substring(7, -1);` can do the same.

Reply:
Hello, FedorovD! From developer.mozilla: "Any argument value that is less than 0 ... is treated as if it were 0 ... .". So, for example: substring(-1,2) equals substring(0,2).  And in my case function with substring(7,-1) cannot pass the test.